pvm/ts/src/engine/actions/raiseAction.ts

```
@@ -3,27 +3,28 @@ import { Player } from "../../models/game";
-        
-        if (player.chips < this.getDeductAmount(player, this.game.bigBlind))
-            throw new Error("Player has insufficient chips to raise.");
 
-        return { minAmount: this.game.bigBlind, maxAmount: player.chips };
+        const lastBet = this.game.getLastAction();
+        if (!lastBet) throw new Error("No previous bet to raise.");
+
+        const minAmount = (lastBet?.amount || 0n) + this.game.bigBlind;
+        if (player.chips < minAmount) throw new Error("Player has insufficient chips to raise.");
+
+        const minAmount = (lastBet?.amount || 0n) + this.game.bigBlind;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'minAmount'. Consider to delete it or rename it.

'minAmount' is defined here:


(lint/suspicious/noRedeclare)